### ISSUE TITLE
Replace @types/rdf-js with @rdfjs/types

### ISSUE THREAD

**Issue by jeswr** [hypermedia-app/hydra-validator#122]
Usage of `@types/rdf-js` should be `@rdfjs/types` instead. See https://github.com/rdfjs/types?tab=readme-ov-file#what-about-typesrdf-js

This was discovered in the same way as https://github.com/jacoscaz/quadstore/issues/167#issuecomment-1930703128.

https://github.com/hypermedia-app/hydra-validator/blob/b520a80133a1ed8c17cd20886b91edcd15c090de/validator-analyse/package.json#L37

**Comment by tpluscode** [comment]
Curious, are you automatically generating this and similar issues? Because you dug up some real old repos :)

**Comment by jeswr** [comment]
I was trying to pin down some warnings that I was getting from components.js building the Community Solid Server so I ran the following search to see what uses of @types/rdf-js were still in use

https://github.com/search?q=%22%5C%22%40types%2Frdf-js%5C%22%3A+%5C%22%22+language%3AJSON&type=code

**Comment by tpluscode** [comment]
I got that from the linked issue. I wonder about the technique of creating these issues. There are 600+ matches for `@types/rdf-js` :)

And by the way, we should also discourage usages of `rdf-js`. Another 400+ hits on public GitHub... You probably noticed that I did that recently in DT.

**Comment by jeswr** [comment]
>  I wonder about the technique of creating these issues. There are 600+ matches for @types/rdf-js :)

Aha - I'm just going through the first couple of pages worth of non-archived repos by hand (filtering out forks also brings the number down to 300). Sorry for any noise created on repo's that aren't in use much anymore.

**Comment by jeswr** [comment]
> And by the way, we should also discourage usages of rdf-js. Another 400+ hits on public GitHub... You probably noticed that I did that recently in DT.

Agreed - looks like you have write permissions for https://www.npmjs.com/package/rdf-js; perhaps you could mark it as deprecated (per https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions)

**Comment by tpluscode** [comment]
Good point. Thanks

**Comment by tpluscode** [comment]
Done, and updated the readme (https://github.com/rdfjs/types/pull/40)

I tweaked your search terms a little. In addition to excluding forks, I also excluded archived repos and match only `package.json`. Only 139 hits :)

https://github.com/search?q=%22%5C%22%40types%2Frdf-js%5C%22%3A+%5C%22%22+language%3AJSON++NOT+is%3Aarchived+NOT+is%3Afork+path%3A%2Fpackage.json%24%2F&type=code

And 69 of `rdf-js`

https://github.com/search?q=%22%5C%22rdf-js%5C%22%3A+%5C%22%22+language%3AJSON++NOT+is%3Aarchived+NOT+is%3Afork+path%3A%2Fpackage.json%24%2F&type=code
